Scripts/script-IsMaliciousIndicatorFound.yml

```diff
@@ -3,60 +3,43 @@ commonfields:
   version: -1
 name: IsMaliciousIndicatorFound
 script: |-
-  var files = dq(invContext,'File(val.Malicious)');
-  var urls = dq(invContext,'URL.Malicious');
-  var ips = dq(invContext,'IP(val.Malicious)');
-  var domains = dq(invContext,'Domain(val.Malicious)');
-  var emails = dq(invContext,'Account.Email(val.Malicious)');
+  minScore = (args.includeSuspicious == 'yes') ? 2 : 3
 
-  if (args.includeSuspicious == 'yes') {
-      if (!files || files.length === 0) {
-          files = dq(invContext,"DBotScore(val.Type.toLowerCase()=='file' && val.Score>1)");
-      }
-      if (!ips || ips.length === 0) {
-          ips = dq(invContext,"DBotScore(val.Type.toLowerCase()=='ip' && val.Score>1)");
-      }
-      if (!urls || urls.length === 0) {
-          urls = dq(invContext,"DBotScore(val.Type.toLowerCase()=='url' && val.Score>1)");
-      }
-      if (!domains || domains.length === 0) {
-          domains = dq(invContext,"DBotScore(val.Type.toLowerCase()=='domain' && val.Score>1)");
+  files = dq(invContext,`DBotScore(val.Type.toLowerCase()=='file' && val.Score>=${minScore})`)
```

Comment:
Not sure using `DBotScore` instead of context File/URL/Domain/Account will cover all cases.. and not cause regrassion.
But @anara123 is better to decide on this